invenio_cli/helpers/docker_helper.py

```diff
@@ -57,11 +57,12 @@ def start_containers(self):
 
     def stop_containers(self):
         """Stop currently running containers."""
-        command = ['docker-compose',
-                   '--file', 'docker-compose.full.yml', 'stop']
-
-        if self.local:
-            command[2] = 'docker-compose.yml'
+        command = [
+            'docker-compose',
+            '--file',
+            'docker-compose.yml' if self.local else 'docker-compose.full.yml',
+            'stop'
+        ]
 
         subprocess.call(command)
```

Comment:
Turns out the same trick @lnielsen mentioned can be used for the stop command too (It works. I learned something!): 

```python
def stop_containers(self):
    """Stop currently running containers."""
    command = [
        'docker-compose',
        '--file',
        'docker-compose.full.yml',
        'stop'
    ]
    subprocess.call(command)
```


Testing becomes much easier and uniform too.